apps/web/src/server/views/_includes/macros/make-appeal-site-html.njk

```diff
@@ -0,0 +1,12 @@
+{% macro makeAppealSiteHtml(appealSiteData) %}
```

Comment:
Should we wrap the entire thing in an <address> tag to make it more semantic?

Reply:
I've removed this macro entirely as it's obsolete now (using `address` instead).